config.json

```diff
@@ -1596,8 +1596,8 @@
       {
         "groupId": "com.google.android.libraries.places",
         "artifactId": "places-compat",
-        "version": "2.2.0",
-        "nugetVersion": "2.2.0",
+        "version": "2.1.0",
```

Comment:
Version downgrade

Reply:
I specifically asked to review and merge this PR as 1st (or after last GPS one)

From discord:

> is there any way to commit #730 and #731 first?  the issue is that #729 also contains the changes in the other 2